README.md

```diff
@@ -34,6 +34,16 @@ features = ["yaml_conf"]
 default-features = false
 ```
 
+## Using ron
```

Comment:
This section needs a bit more clarity; any feature for `confy` is at odds with another, so you cannot have `RON` and `YAML`, or `TOML` and `RON`, or `TOML` and `YAML`. 

Can you please update this section to clarify that?

Reply:
Resolved in https://github.com/rust-cli/confy/pull/71/commits/e077f3c1d4bfe8c93b9529d56040e10f9576cf6e. I tried my best explaining that they conflict with each other and how to enable the different features, but let me know if I can make any improvements.